x/twap/logic_test.go

```diff
@@ -666,15 +726,15 @@ func (s *TestSuite) TestPruneRecords() {
 
 	pool1OlderMin2MsRecord, // deleted
 		pool2OlderMin1MsRecordAB, pool2OlderMin1MsRecordAC, pool2OlderMin1MsRecordBC, // deleted
-		pool3OlderBaseRecord, // kept as newest under keep period
+		pool3OlderBaseRecord,    // kept as newest under keep period
 		pool4OlderPlus1Record := // kept as newest under keep period
```

Comment:
these both cant be kept as newest right?

Reply:
this diff comes from simple move of code, still changing the comment tho